core/src/net/sf/openrocket/aerodynamics/BarrowmanCalculator.java

```diff
@@ -667,8 +671,10 @@ private double calculateBaseDrag(FlightConfiguration configuration, FlightCondit
 				
 				// if I'm the last component, set my base CD
 				// note:  the iterator *should* serve up the next component.... buuuut ....
-				//        this code has is tested, and there's no compelling reason to change. 
-				if (s.getNextSymmetricComponent() == null) {
+				//        this code has is tested, and there's no compelling reason to change.
+				final SymmetricComponent n = s.getNextSymmetricComponent();
+				if ((n == null) ||
+					!configuration.isStageActive(n.getStageNumber())) {
```

Comment:
line 676 -> 677 is misleading indentation :/

Reply:
That's the problem with using tabs for indentation...  just merged a PR (793) to clean it up.

Reply:
Ah, it was a nitpick anyway.  But thanks for cleaning it up ^^